### PR TITLE
Build manifest for org.debian.TuxPuck

### PR DESCRIPTION
**Pull request by cosimoc**
Based on the Freedesktop 1.6 SDK.

### CODE REVIEW

**Comment by Barthalion** [comment]
Build status [here](https://flathub.org/builds/#/buildrequests/849?redirect_to_build=true).

**Comment by Barthalion** [comment]
Merged into https://github.com/flathub/org.debian.TuxPuck

File: org.debian.TuxPuck.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.libsdl.org/release/SDL-1.2.15.tar.gz",
+                    "sha256": "d6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00"
+                },
+                {
+                    "type": "patch",
+                    "path": "sdl-libx11-build.patch"
+                },
```
Comment (alexlarsson) [inline org.debian.TuxPuck.json]: Lots of things seems to use SDL1, and this even has a patch. I wonder if we should add it to shared-modules.

**Comment by cosimoc** [comment]
Yeah, it's used by a lot of modules. I would love to split it out to a shared module.

**Pull request closed without merging**